components/DeleteButton.js

```diff
@@ -1,21 +1,27 @@
 import styled from "styled-components";
 
 export default function DeleteButton({ children, handleClick, id }) {
```

Comment:
You should only pass two props to this component (children and handleClick). You can pass the function () => handleClick(id) to this component, so the id is not needed here.